docs/resources/dhcp_host.md

```diff
@@ -0,0 +1,82 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_dhcp_host Resource - terraform-provider-bloxone"
+subcategory: "IPAM/DHCP"
+description: |-
+  Manages DHCP Hosts.
+  A DHCP Host object associates a DHCP Config Profile with an on-prem host.
```

Comment:
I would another  line here that says "Note: This resource represents an existing backend object that cannot be created or deleted through API calls. Instead, it can only be updated. When using `terraform apply` the resource configuration is applied to the existing object, and no new object is created. Similarly `terraform destroy` removes the configuration associated with the object without actually deleting it from the backend."

Reply:
Note Added